home.html

```diff
@@ -31,13 +31,162 @@ <h2>I'm a software developer! I can help you build a product, feature or website
+            <li class="tonic_items"> 2015 </li>
+          </ul>
+        </div>
+        <div class="daily_selection">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>>
+        </div>
+        <div class="languajes">
+          <ul class="languajes_container">
+            <li class="box_languaje"> html </li>
+            <li class="box_languaje"> css </li>
+            <li class="box_languaje">JavasScript</li>
+          </ul>
+        </div>
+        <button type="button" class="project">See project</button>
+      </div>
+    </div>
+  </section>
+  <section id="about_me">
+    <div class="about_container">
+      <div class="about_myself">
+        <h1>About<br>
+          Myself</h1>
+      </div>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3 class="connect">
+        LET'S CONNECT
+      </h3>
       <ul class="social_media">
-        <li class="social_icon"> <a href="https://twitter.com/CruzGrebi"><img alt="twitter" src="image/twitter.svg"></a></li>
+        <li class="social_icon"> <a href="https://twitter.com/CruzGrebi"><img alt="twitter" src="image/twitter.svg"></a>
+        </li>
         <li class="social_icon"> <a href="#"><img alt="letterm" src="image/vectorm.svg"></a></li>
-        <li class="social_icon"> <a href="https://www.linkedin.com/in/grebli-cruz-443004233"><img alt="linkedin" src="image/Linkedin.svg"></a></li>
-        <li class="social_icon"> <a href="https://github.com/GrebliCruz"><img alt="github" src="image/github.svg"></a></li>
+        <li class="social_icon"> <a href="https://www.linkedin.com/in/grebli-cruz-443004233"><img alt="linkedin"
+              src="image/Linkedin.svg"></a></li>
+        <li class="social_icon"> <a href="https://github.com/GrebliCruz"><img alt="github" src="image/github.svg"></a>
+        </li>
         <li class="social_icon"> <a href="#"><img alt="peacehand" src="image/vectorhand.svg"></a></li>
+      </ul>
+  
+    <button type="button" class="project">Get my resume </button>
+   <div class="langguages_box"> 
+         <ul class="langguages_container"> 
+           <li class="list_langguages">Langguages
+             <ol>
+                <li class="langguage_items"> <img alt="JS" src="image/Ellipse1.png">JavasScript</li>
+                <li class="langguage_items"> <img alt="HTML" src="image/Ellipse2.png">HTML</li>
+                <li class="langguage_items"> <img alt="CSS" src="image/Ellipse3.png">CSS</li>
+             </ol>
+           <li class="list_langguages">Frameworks <img class="arrow" alt="dropdown"  src="image/arrow.png"></li>
+           <li class="list_langguages">Skills <img class="arrow" alt="dropdown" src="image/arrow.png"></li>
```

Comment:
- [ ] Great job implementing the arrows as requested by the last reviewer! However, we need to add some styling so it matches the Figma design:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/159146894-75c7b1ef-7817-4d6d-9f7c-2282e3420c06.png) | ![image](https://user-images.githubusercontent.com/57421823/159146890-dd3fa7ae-982f-48b0-9bed-4ee70dda6120.png)

Tips:

- Check the margins/paddings
- Check the font styling to match
- The top line shouldn't be there (there should be only 2 lines).

That way, it will look just like the Figma! 💪